src/main/java/bio/terra/cli/utils/FileUtils.java

```diff
@@ -63,4 +67,65 @@ public static Path writeStringToFile(File outputFile, @Nullable String fileConte
 
     return Files.writeString(outputFile.toPath(), fileContents == null ? "" : fileContents);
   }
+
```

Comment:
This method is moved from `TestContext` to here so it can be shared with `MountController`